discovery.md

```diff
@@ -21,33 +21,39 @@ version.
 
 ## Overview
 
-The goal of the CloudDiscovery API is to enable connections between consumers of
-events and Event producers to facilitate the creation of a
-CloudSubscription. Note that whether an Event producer is a single entity,
-an aggregator/broker of multiple event producers, or an intermediary is an
-implementation detail of the producer and does not change how an event
-consumer interacts with a specification compliant producer.
-
-Discovery allows for an event producer to
-advertise the event types that are available, provide the necessary information
-to consume that event (schema / delivery protocol options), and the necessary
-information to create a subscription. The output of the API ought to be such
-that tooling can be built where all possible event producers and types aren’t
-known in advance.
+In order for consumers to receive events from event producers, they need
+to first subscribe, or ask for, events from those producers. To do so, there
+is often a process necessary that involves steps such as discovering which
+event producer is of interest, what events it generates and how to create the
+subscription for those events.
```

Comment:
I struggled with this because in CE we use, and define, the term "producer" to mean the entity that creates the CE - more as a piece of middleware that's acting as a serialization engine. But, then there are times when it makes sense to use the term "event producer" to mean "something that generates events" - the more generic meaning outside of CE. And I can try to use the term "service" but it might feel awkward at time.... but we'll see....